packages/components/molecules/f-card-with-content/src/components/CardWithContent.vue

```diff
@@ -1,32 +1,107 @@
 <template>
-    <div
-        :class="$style['c-cardWithContent']"
-        data-test-id="cardWithContent">
-        I am a CardWithContent Component (GB)
-    </div>
+    <card-component
+        has-outline
+        is-page-content-wrapper
+        data-test-id="cardWithContent"
+        :class="$style['c-cardWithContent']">
+        <div
+            v-if="$slots.icon"
+            :class="$style['c-cardWithContent-icon']">
+            <slot name="icon" />
+        </div>
+
+        <h1
+            v-if="cardHeading"
+            :class="$style['c-cardWithContent-heading']">
+            {{ cardHeading }}
+        </h1>
+
+        <p
+            v-if="cardDescription"
+            :class="$style['c-cardWithContent-description']">
+            {{ cardDescription }}
+        </p>
+
+        <f-button
+            v-if="primaryButtonText"
+            button-size="large"
+            button-type="primary"
+            is-full-width
+            :class="$style['c-cardWithContent-button']"
+            @click.native="$emit('primary-button-click')">
+            {{ primaryButtonText }}
+        </f-button>
+
+        <f-button
+            v-if="secondaryButtonText"
+            button-size="large"
+            button-type="secondary"
+            is-full-width
+            :class="$style['c-cardWithContent-button']"
+            @click.native="$emit('secondary-button-click')">
+            {{ secondaryButtonText }}
+        </f-button>
+    </card-component>
 </template>
 
 <script>
+import CardComponent from '@justeat/f-card';
+import '@justeat/f-card/dist/f-card.css';
+import FButton from '@justeat/f-button';
+import '@justeat/f-button/dist/f-button.css';
 
 export default {
     name: 'CardWithContent',
-    components: {},
+    components: {
+        CardComponent,
+        FButton
+    },
     props: {
+        cardHeading: {
+            type: String,
+            default: ''
+        },
+        cardDescription: {
+            type: String,
+            default: ''
+        },
+        primaryButtonText: {
+            type: String,
+            default: ''
+        },
+        secondaryButtonText: {
+            type: String,
+            default: ''
+        }
     }
 };
 </script>
 
 <style lang="scss" module>
-
 .c-cardWithContent {
     display: flex;
-    justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
-    margin: auto;
-    border: 1px solid red;
-    font-family: $font-family-base;
-    @include font-size(heading-m);
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+}
+
+.c-cardWithContent-heading {
```

Comment:
Not sure if this will be the case here, but usually headings are slightly different sizes on narrow compared with wide viewport widths. So may need to define that here too (the font-map usually has two values, so if this is the wide font-size, you can do `@include font-size(heading-s, 'true', 'narrow');` (or something v similar to that to output the narrow size)

Reply:
These styles were lifted straight out of f-checkout, which already works pretty well. The fact the heading is already small means it doesn't look strange when the viewport gets narrow.

Reply:
The narrow/wide headings are what’s set in PIE, so they should be used wherever those headings are used (as I understand it). Don’t want to tie this ticket to it, but this should be fixed so if make a ticket to check this with design and fix it if it is an issue

Reply:
Have updated to match the designs (which incidentally also needed to be updated 😄)